pkg/operator/sync_openshiftcontrollermanager_v311_00.go

```diff
@@ -143,3 +160,78 @@ func manageOpenShiftControllerManagerPublicConfigMap_v311_00_to_latest(client co
 
 	return resourceapply.ApplyConfigMap(client, configMap)
 }
+
+func manageBuildAdditionalCAConfigMap(client coreclientv1.ConfigMapsGetter, configClient configclientv1.BuildsGetter, operatorConfig *v1alpha1.OpenShiftControllerManagerOperatorConfig) (string, bool, error) {
+	caMap := resourceread.ReadConfigMapV1OrDie(v311_00_assets.MustAsset("v3.11.0/openshift-controller-manager/build-additional-ca-cm.yaml"))
+	buildConfig, err := configClient.Builds().Get("cluster", metav1.GetOptions{})
+	if err != nil && !errors.IsNotFound(err) {
+		return "", false, err
+	}
+	if buildConfig == nil {
+		caMap.Data = nil
+		_, modified, err := resourceapply.ApplyConfigMap(client, caMap)
+		operatorConfig.Status.AdditionalTrustedCA = nil
+		return "", modified, err
+	}
+
+	caData, err := mergeCAConfigMap(client, buildConfig.Spec.AdditionalTrustedCA)
+	if err != nil {
+		return "", false, err
+	}
+	if len(caData) == 0 && operatorConfig.Status.AdditionalTrustedCA != nil {
+		caMap.Data = nil
+		_, modified, err := resourceapply.ApplyConfigMap(client, caMap)
+		operatorConfig.Status.AdditionalTrustedCA = nil
+		return "", modified, err
+	}
+	h := sha1.New()
+	h.Write(caData)
+	caHash := fmt.Sprintf("%x", h.Sum(nil))
+	if operatorConfig.Status.AdditionalTrustedCA != nil &&
+		operatorConfig.Status.AdditionalTrustedCA.SHA1Hash == caHash {
+		return caHash, false, nil
+	}
+	operatorConfig.Status.AdditionalTrustedCA = &v1alpha1.AdditionalTrustedCA{
+		SHA1Hash:      caHash,
+		ConfigMapName: buildConfig.Spec.AdditionalTrustedCA.Name,
+	}
+	caMap.Data["additional-ca.crt"] = string(caData)
+	_, modified, err := resourceapply.ApplyConfigMap(client, caMap)
+	return caHash, modified, err
```

Comment:
If updates are happening, copy the merged data into the ConfigMap, which is then mounted into the build controller